src/main/java/io/github/thebusybiscuit/slimefun4/utils/biomes/BiomeMap.java

```diff
@@ -150,13 +151,34 @@ public String toString() {
 
     @ParametersAreNonnullByDefault
     public static <T> @Nonnull BiomeMap<T> fromResource(NamespacedKey key, String path, BiomeDataConverter<T> valueConverter) throws BiomeMapException {
+        return fromResource(key, Slimefun.instance(), path, valueConverter);
+    }
```

Comment:
I completely forgot that different resources will need a different `JavaPlugin` instance. :NotLikeThis:

In my opinion, we can just drop this method and pass `Slimefun.instance()` for our own resources, it'd be too tempting to expose this method here.

Reply:
Agreed